tests/baselines/reference/spreadExcessProperty.errors.txt

```diff
@@ -0,0 +1,10 @@
+spreadExcessProperty.ts(3,7): error TS2353: Object literal may only specify known properties, and 'extra' does not exist in type 'A'.
+
+
+==== spreadExcessProperty.ts (1 errors) ====
+    type A = { a: string, b: string };
+    const extra1 = { a: "a", b: "b", extra: "extra" };
+    const a1: A = { ...extra1 }; // spread should not give excess property errors

```

Comment:
The referenced issue is primarily about object literals that are spread. This is a situation in which an existing object type is spread *into* an object literal and that's a different situation.

Reply:
Yes. Though some of the comments were asking for more, so I am experimenting